src/components/network-modification-table/renderers/root-network-chip-cell.tsx

```
@@ -0,0 +1,143 @@
+            };
+        });
+    } else {
+        updateStatus(false);
+        return [
+            ...prev,
+            {
+                rootNetworkUuid,
+                modificationUuidsToExclude: [modificationUuid],
+            },
+        ];
+    }
```

ESLint

[error] Unnecessary 'else' after 'return'.

(no-else-return)